src/tests/utils/ApiTestSteps.ts

```diff
@@ -30,116 +30,147 @@ const xmlParser = new XMLParser();
 const ajv = new Ajv({ strictTuples: false });
 
 export async function startStubServiceAndReturnSessionId(): Promise<any> {
-	const stubResponse = await stubStartPost();
+	const stubResponse = await stubStartPostByType("FACE_TO_FACE");
 	return sessionPost(stubResponse.data.clientId, stubResponse.data.request);
 }
 
-export async function stubStartPost():Promise<any> {
+export async function startStubServiceAndReturnSessionIdByType(journeyType: string): Promise<any> {
+	const stubResponse = await stubStartPostByType(journeyType);
+	return sessionPost(stubResponse.data.clientId, stubResponse.data.request);
+}
+
+export async function stubStartPost(): Promise<any> {
+	return stubStartPostByType("FACE_TO_FACE");
+}
+
+export async function stubStartPostByType(journeyType: string): Promise<any> {
 	const path = constants.DEV_IPV_STUB_URL;
-	const postRequest = await axios.post(`${path}`);
+
+	let postRequest: AxiosResponse;
+	switch (journeyType) {
+		case "NO_PHOTO_ID":
+			postRequest = await axios.post(`${path}`, { context: "bank_account" });
+			break;
+		case "INVALID":
+			postRequest = await axios.post(`${path}`, { context: "INVALID" });
+			break;
+		default:
+			postRequest = await axios.post(`${path}`);
+			break;
+	}
+
 	expect(postRequest.status).toBe(201);
 	return postRequest;
 }
 
-export async function sessionPost(clientId?: string, request?: string):Promise<any> {
+export async function sessionPost(clientId?: string, request?: string): Promise<any> {
 	const path = "/session";
 	try {
 		const postRequest = await API_INSTANCE.post(path, { client_id: clientId, request });
-		expect(postRequest.status).toBe(200);
 		return postRequest;
 	} catch (error: any) {
 		console.log(`Error response from ${path} endpoint: ${error}`);
 		return error.response;
-	} 
+	}
+}
+
+export async function sessionConfigGet(sessionId: string) {
```

Comment:
Return type plssssss

Reply:
I wonder if I will ever remember to add that 🙄

Reply:
😂 you can set up prettier in vscode so that it tells you (I can never remember either)